### DISCUSSION TITLE
Run Actions from Cards

### DISCUSSION THREAD

**Discussion by goldmerc** [laravel/nova-issues#4642] [Q&A]
Hi, I'd like to be able to have a button on a card to run a standalone action. Ideally, I'd like to be able to have the card on a dashboard rather than a resource page.

I've made some custom cards and fields before but I'm a beginner with vue and I'd be very grateful if anyone could point me in the right direction on how to use the existing action modal component from inside a custom card?

Many thanks!

**Comment by abordage** [comment]
You can see how the buttons are implemented here: https://github.com/stepanenko3/nova-cards

**Comment by goldmerc** [comment]
Thanks @abordage - that got me on the right track.

I realised that I don't need to use a Nova action at all. Just a button to run some php on the back end.

Got it working nicely with a confirmation modal by looking at the 'cache card' from the package you recommended.

Thanks!
